rust_snuba/src/processors/release_health_metrics.rs

```diff
@@ -0,0 +1,761 @@
+use adler::Adler32;
+use anyhow::Context;
+use chrono::{Date, DateTime, Utc};
+use core::f64;
+use rust_arroyo::backends::kafka::types::KafkaPayload;
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeMap;
+
+use crate::{
+    types::{InsertBatch, RowData},
+    KafkaMessageMetadata, ProcessorConfig,
+};
+
+use super::utils::enforce_retention;
+
+const GRANULARITY_TEN_SECONDS: u8 = 0;
+const GRANULARITY_ONE_MINUTE: u8 = 1;
+const GRANULARITY_ONE_HOUR: u8 = 2;
+const GRANULARITY_ONE_DAY: u8 = 3;
```

Comment:
These are probably not used in the release health pipeline. So you can get rid of them

Reply:
Fixed, thanks